Modules/DataModel/Sources/PocketCastsDataModel/Public/DataManager.swift

```diff
@@ -98,6 +98,15 @@ public class DataManager {
         return convertedEpisodes
     }
 
+    public func allUpNextEpisodeUuids() -> [BaseEpisode] {
+        upNextManager.allUpNextPlaylistEpisodes(dbQueue: dbQueue).map{
```

Comment:
I thought here we would use the new UUID methods you added on the DataManager?

Reply:
My bad. Turns out, we don't need them! I realized it at the last minute and I forgot to delete the new methods. The list of UUIDs can be fetched by just calling the `upNextManager` table, no need to fetch from the `episodes` and `user_episodes` tables. The only "problem" is that all the "anemic" objects will be of type `Episode`, even if they really are representing UserEpisodes. That isn't an issue with the current implementation of `EpisodeViewModel` though.